pkg/features/kube_features.go

```diff
@@ -904,6 +905,11 @@ const (
 
 func init() {
 	runtime.Must(utilfeature.DefaultMutableFeatureGate.Add(defaultKubernetesFeatureGates))
+
+	// replace client-go's feature gate instance with kube's
```

Comment:
explain why, not just what.  This wires the kube binary CLI flags as the authoritative source of the client-go featuregates.